### PR TITLE
f_4_login_signup_pages

### PR DESCRIPTION
**Pull request by Greengrer**
- Added login.html and signup.html templates.
- Added signup, login, and common css files.
- Added a font for input placeholders.

### CODE REVIEW

**Review by nomarchia — APPROVED** [review]

File: src/main/webapp/WEB-INF/templates/login.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+				<header class="header">
+					<div class="app-name">App-name</div>
+					<div class="navigation-row">
+						<a href="#" class="navigation-link">Home</a>
+						<a href="#" class="navigation-link">Sign Up</a>
+					</div>
+				</header>
+					<div class="body__column">
+						<form action="" class="profile-text">
+							<input type="text" id="username" class="styled-as-input" name="username" placeholder="put your username here">
```
Comment (AlexKievDeveloper) [inline src/main/webapp/WEB-INF/templates/login.html:23]: Please change name="username" --> name="login"

Comment (AlexKievDeveloper) [inline src/main/webapp/WEB-INF/templates/login.html:23]: row 23 Please change name="username" to name="login"

**Review by RomanKlishch — APPROVED** [review]

**Review by AlexKievDeveloper — APPROVED** [review]

File: src/main/webapp/WEB-INF/templates/signup.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+					<div class="navigation-row">
+						<a href="#" class="navigation-link">Home</a>
+						<a href="#" class="navigation-link">Sign Up</a>
+					</div>
+				</header>
+					<div class="body__column">
+						<form action="" class="registration-text">
+							<div class="text-information__row">
+							<div class="hidden-information__column">
+									<input type="text" id="e-mail" class="styled-as-input" name="e-mail" placeholder="put your e-mail here">
```
Comment (AlexKievDeveloper) [inline src/main/webapp/WEB-INF/templates/signup.html:25]: row 25 <input type = "email">

**Review by AlexKievDeveloper — APPROVED** [review]

File: src/main/webapp/WEB-INF/templates/login.html (hunk 1 above)
Comment (Greengrer) [reply src/main/webapp/WEB-INF/templates/login.html]: Done, thank you!

File: src/main/webapp/WEB-INF/templates/signup.html (hunk 2 above)
Comment (Greengrer) [reply src/main/webapp/WEB-INF/templates/signup.html]: Done, thank you!

File: src/main/webapp/WEB-INF/templates/change-password.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Create a greeting card!</title>
+		<meta name="viewport" content="width=device-width, initial-scale=0.86, maximum-scale=3.0, minimum-scale=0.86" />
+		<link rel="stylesheet" href="../../assets/css/reset.css" />
+		<link rel="stylesheet" href="../../assets/css/common.css" />
+		<link rel="stylesheet" href="../../assets/css/change-password.css" />
+		<meta charset="UTF-8">
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/change-password.html:9]: Common links must be in common file. It uses in all html pages.

File: src/main/webapp/WEB-INF/templates/change-password.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+		<div class="wrapper">
+			<div class="main-body">
+				<header class="header">
+					<div class="app-name">App-name</div>
+					<div class="navigation-row">
+						<a href="#" class="navigation-link">Home</a>
+						<a href="#" class="navigation-link">Your cards</a>
+						<a href="#" class="navigation-link">Profile</a>
+						<a href="#" class="navigation-link">Log out</a>
+					</div>
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/change-password.html:21]: Navigation menu must be in common file. Nav menu uses in all pages.

File: src/main/webapp/WEB-INF/templates/change-password.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+							<input type="password" id="old-password" class="styled-as-input" name="password" placeholder="put your old password here">
+							<input type="password" id="new-password" class="styled-as-input" name="new-password" placeholder="put your new password here">
+							<input type="password" id="new-password-check" class="styled-as-input" name="new-password-check" placeholder="and here, to check">
+							<input type="submit" class="button-link" value="Log in">
+						</form>
+					</div>
+				<footer class="footer">
+					<div>Some words</div>
+					<div>Copyright, 2020</div>
+				</footer>
```
Comment (OleksandrGubariev) [inline src/main/webapp/WEB-INF/templates/change-password.html:34]: Footer must be in common file. Footer uses in all pages.

Comment (Greengrer) [reply src/main/webapp/WEB-INF/templates/change-password.html:34]: Noted! As soon as I understand how to change it right for the thymeleaf I am changing it.

**Review by RomanKlishch — APPROVED** [review]

**Pull request closed without merging**